content/en/blog/2025/infernece-extension-support/index.md

```diff
@@ -0,0 +1,155 @@
+
+Additionally, many modern AI deployments use fine-tuned adapters like [LoRA](https://arxiv.org/abs/2106.09685) (Low-Rank Adaptation) to customize model behavior for specific users, organizations, or use cases. These adapters consume GPU memory and loading time when switched. A model server that already has the required LoRA adapter loaded can process requests immediately, while servers without the adapter face expensive loading overhead that can take seconds to complete.
+
+### **Queue Dynamics and Criticality**
+
+AI inference workloads also introduce the concept of request criticality that's less common in traditional services. Real-time interactive applications (like chatbots or live content generation) require low latency and should be prioritized, while batch processing jobs or experimental workloads can tolerate higher latency or even be dropped during system overload.
+
+Traditional load balancers lack the context to make these criticality-based decisions. They can't distinguish between a time-sensitive customer support query and a background batch job, leading to suboptimal resource allocation during peak demand periods.
+
+This is where inference-aware routing becomes critical. Instead of treating all backends as equivalent black boxes, we need routing decisions that understand the current state and capabilities of each model server, including their queue depth, memory utilization, loaded adapters, and ability to handle requests of different criticality levels.
+
+## **Gateway API Inference Extension: A Kubernetes-Native Solution**
+
+The [Kubernetes Gateway API Inference Extension](https://gateway-api-inference-extension.sigs.k8s.io/guides/#__tabbed_3_2) has introduced solutions to these challenges, building on the proven foundation of Kubernetes Gateway API while adding AI-specific intelligence. Rather than requiring organizations to patch together custom solutions or abandon their existing Kubernetes infrastructure, the extension provides a standardized, vendor-neutral approach to intelligent AI traffic management.
+
+The extension introduces two key Custom Resource Definitions that work together to address the routing challenges we've outlined. The InferenceModel resource provides an abstraction for AI-Inference workload owners to define logical model endpoints, while the InferencePool resource gives platform operators the tools to manage backend infrastructure with AI workload awareness.
+
+By extending the familiar Gateway API model rather than creating an entirely new paradigm, the inference extension enables organizations to leverage their existing Kubernetes expertise while gaining the specialized capabilities that AI workloads demand. This approach ensures that teams can adopt intelligent inference routing aligned with familiar networking knowledge and tooling.
+
+Note: InferenceModel is likely to change in future Gateway API Inference Extension releases.
```

Comment:
How?  Enough to make it worth calling out specifically?

(If yes, [use a callout](https://istio.io/latest/docs/releases/contribute/shortcodes/#callouts))